src/featureflagservice/rel/overlays/bin/migrate

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
These scripts are what are used to start the Release. See the command at the end of the dockerfile. Which I now see uses `CMD` instead of `ENTRYPOINT` for some reason... may change that.